examples/repl.rs

```diff
@@ -46,6 +50,10 @@ mod example {
         Locals: (),
         /// Print this help
         Help|h: (),
+        /// Inject and run clif ir
+        Inject: PathBuf,
```

Comment:
If this will only support clif ir, should we name the command as InjectCliff or something to be clear ?